bin/hdfcoinc/pycbc_multiifo_exclude_zerolag

```diff
@@ -0,0 +1,90 @@
+#!/bin/env python
+"""
+Remove all coincs in background_exc that contain triggers at time of zerolag
+coincidences from *any* coincidence type with ifar above a certain threshold
+"""
+
+import h5py, numpy as np, argparse, logging, pycbc, pycbc.events, pycbc.io
+import pycbc.version
+import pycbc.conversions as conv
+from pycbc.events import veto, coinc
+from ligo import segments
```

Comment:
can you remove unneeded imports?  I think pycbc.events and pycbc.conversions are not used at least.